src/Assets/TestProjects/BlazorHosted/blazorhosted/blazorhosted.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
when we have a new runtime and dropping the older runtime in test set up, these test would fail. We'd like ASPNET team to help the transition. Mostly search replace net5.0 to net6.0

Reply:
Sounds sensible. BTW, is there a build property that can be used to sub in here?

Reply:
We don't have a good solution yet. So far we just manually change it. You can try some solutions